pkg/packagekit/updates.jsx

```diff
@@ -471,6 +532,42 @@ class OsUpdates extends React.Component {
             ex => this.handleLoadError((ex.problem == "not-found") ? _("PackageKit is not installed") : ex));
     }
 
+    loadHistory() {
+        let history = [];
+
+        // would be nice to filter only for "update-packages" role, but can't here
+        pkTransaction("GetOldTransactions", [0], {
+                Transaction: (objPath, timeSpec, succeeded, role, duration, data) => {
+                    if (role !== PK_ROLE_ENUM_UPDATE_PACKAGES)
+                        return;
+                    // data looks like:
+                    // downloading	bash-completion;1:2.6-1.fc26;noarch;updates-testing
+                    // updating	bash-completion;1:2.6-1.fc26;noarch;updates-testing
+                    let pkgs = {"_time": Date.parse(timeSpec)};
+                    let empty = true;
+                    data.split("\n").forEach(line => {
+                        let fields = line.split("\t");
+                        if (fields.length >= 2) {
+                            let pkgId = fields[1].split(";");
+                            pkgs[pkgId[0]] = pkgId[1];
+                            empty = false;
+                        }
+                    });
+                    if (!empty)
+                        history.unshift(pkgs);
```

Comment:
Is it a specific optimization to use `unshift` here instead of `push`, i.e. does the order matter? Just curious, I don't think the performance make a difference here. :)